Lib/asyncio/tasks.py

```diff
@@ -639,10 +639,34 @@ def __sleep0():
     """
     yield
 
+def __check_delay(delay):
+    """Check if the value of 'delay' is valid."""
+
+    import sys
+    import math
+
+    SEC_TO_NS = 1000 * 1000 * 1000
+
+    if not (isinstance(delay, int) or isinstance(delay, float)):
+        raise TypeError(f"'{type(delay)} object cannot be interpreted as an integer'")
```

Comment:
```suggestion
        raise TypeError(f"{type(delay)} object cannot be interpreted as an integer")
```
There's no need for single quotes.

Reply:
Ok
Thank you